connectors/migrations/20241218_backfill_gdrive_shared_with_me.ts

```diff
@@ -0,0 +1,37 @@
+import { makeScript } from "scripts/helpers";
+
+import { GOOGLE_DRIVE_SHARED_WITH_ME_VIRTUAL_ID } from "@connectors/connectors/google_drive/lib/consts";
+import { getInternalId } from "@connectors/connectors/google_drive/temporal/utils";
+import { dataSourceConfigFromConnector } from "@connectors/lib/api/data_source_config";
+import { concurrentExecutor } from "@connectors/lib/async_utils";
+import { upsertDataSourceFolder } from "@connectors/lib/data_sources";
+import { ConnectorResource } from "@connectors/resources/connector_resource";
+
+makeScript({}, async ({ execute }, logger) => {
+  const connectors = await ConnectorResource.listByType("google_drive", {});
+
+  await concurrentExecutor(
+    connectors,
+    async (connector) => {
+      const folderId = getInternalId(GOOGLE_DRIVE_SHARED_WITH_ME_VIRTUAL_ID);
```

Comment:
just checking: you told me documents already have this folderId in their parents right? (it was just not pointing to any real folder until this PR)

Reply:
actually we never pass this as a parentId, I kept this PR iso-behavior with main but I can't figure out how it used to work. I'll check again and fix it if needed